src/main/java/thePackmaster/cards/warlockpack/Imp.java

```diff
@@ -30,7 +30,6 @@ public Imp() {
         super(ID, COST, AbstractCard.CardType.SKILL, AbstractCard.CardRarity.SPECIAL, AbstractCard.CardTarget.ALL_ENEMY, CardColor.COLORLESS);
         magicNumber = baseMagicNumber = DAMAGE;
         secondDamage = baseSecondMagic = DAMAGE_INCREASE;
-        AutoplayField.autoplay.set(this, true);
```

Comment:
This was just cleaning up something that was redundant and confusing my debugging -- the patches I wrote handle everything Autoplay does, and having the autoplay set was redundant (I noticed it because it was trying to add Imps to the action queue twice, but the second time it would detect that the card was already in the queue).